src/symtab/symbol.hpp

```diff
@@ -17,6 +17,7 @@
 
 #include "lexer/modtoken.hpp"
 #include "symtab/symbol_properties.hpp"
+#include <ast/ast_decl.hpp>
```

Comment:
just a nitpick: in order to prefer/prioritizes headers in the current working (than system dirs), we have been using `""` than `<>`. I think Tristan cleaned that in the past.